bin/MadGraph5_aMCatNLO/cards/production/13TeV/WPrime_tb/WPrime_tb_customizecards.dat

```diff
@@ -0,0 +1,6 @@
+set param_card MASS 34 5.00000e+02

```

Comment:
Is this only mass point that you will be generating?
If there are other mass points can you also upload a card-generating script?

Reply:
Just pushed a commit with a generating script